ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -372,7 +372,7 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
                                     />
                                 </div>
                                 <div className='row small-11'>
-                                    <div className='columns small-3'>NAME</div>
```

Comment:
If we keep here to 3, we get the following cols.
![image](https://github.com/argoproj/argo-workflows/assets/83329336/9b3196d7-76dc-4908-9c02-513d7ef9d56c)